### DISCUSSION TITLE
sync zen mods

### DISCUSSION THREAD

**Discussion by mirsella** [zen-browser/desktop#3455] [Ideas]
hello

would be cool to have the installed zen mods sync like browser extension, instead of having to find them again, configure them again on each computer.

i don't think it's possible by directly using the mozilla account for storage ?
i don't know the hosting plan for the project, but maybe a simple db or a free plan on cloud provider should suffice for such low data i think.
i've also seen extension like stylus or tampermonkey use google drive as storage for sync.

thanks !

**Comment by Eveeifyeve** [comment]
Duplicate: https://github.com/zen-browser/desktop/discussions/1956

**Reply by mirsella** [reply]
oh sorry the search didn't bring it up :(
